src/main/java/net/wurstclient/hacks/AutoTranslateHack.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 - 2019 | Wurst-Imperium | All rights reserved.
+ *
+ * This source code is subject to the terms of the GNU General Public
+ * License, version 3. If a copy of the GPL was not distributed with this
+ * file, You can obtain one at: https://www.gnu.org/licenses/gpl-3.0.txt
+ */
+package net.wurstclient.hacks;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+
+import net.minecraft.client.gui.hud.ChatHud;
+import net.minecraft.text.LiteralText;
+import net.wurstclient.Category;
+import net.wurstclient.SearchTags;
+import net.wurstclient.events.ChatInputListener;
+import net.wurstclient.events.ChatOutputListener;
+import net.wurstclient.hack.Hack;
+import net.wurstclient.settings.CheckboxSetting;
+import net.wurstclient.settings.EnumSetting;
+import net.wurstclient.services.GoogleService;
+import net.wurstclient.services.GoogleService.Language;
+import net.wurstclient.util.ChatUtils;
+
+@SearchTags({ "AutoTranslate", "Translate" })
+public final class AutoTranslateHack extends Hack implements ChatInputListener, ChatOutputListener {
+
+    public static GoogleService google = new GoogleService();;
+
+    private final CheckboxSetting mychat = new CheckboxSetting("Translate your own chat.", false);
+
+    private final CheckboxSetting otherchat = new CheckboxSetting("Translate other chat.", true);
+
+    private final EnumSetting<Language> translate_from = new EnumSetting<>("Translate other chat from?",
+            Language.values(), Language.AUTO_DETECT);
+
+    private final EnumSetting<Language> translate_to = new EnumSetting<>("Translate other chat to?", Language.values(),
+            Language.ENGLISH);
+
+    private final EnumSetting<Language> translate_my = new EnumSetting<>("Translate my own chat to?", Language.values(),
+            Language.JAPANESE);
+
+    public AutoTranslateHack() {
+        super("AutoTranslate", "Translate every chat into other language.\n" 
+        + "Translate your message into other language.\n"
+        +"\n"
+        +"Created by Dj-jom2x");

```

Comment:
I would like to merge it without this line, since hack descriptions aren't meant for credits. You would of course still be mentioned in the changelog and added to the [list of contributors](https://github.com/Wurst-Imperium/Wurst7/graphs/contributors). Would that be OK with you?

Reply:
remove the credits don't worry I'm just playing around

Reply:
do I have to click the resolve conversation button below?